scaling_performance/cns_guidance.adoc

```diff
@@ -0,0 +1,82 @@
+[[scaling-performance-cns-guidance-for-databases]]
+= CNS Guidance For Databases
```

Comment:
We need to also add this topic to the _topic_map.yml file so that it builds in our doc set.